pkg/operator/revisioncontroller/revision_controller.go

```diff
@@ -285,7 +288,17 @@ func (c RevisionController) createNewRevision(ctx context.Context, recorder even
 	}}
 
 	for _, cm := range c.configMaps {
-		obj, _, err := resourceapply.SyncConfigMap(ctx, c.configMapGetter, recorder, c.targetNamespace, cm.Name, c.targetNamespace, nameFor(cm.Name, revision), ownerRefs)
+		obj, _, err := resourceapply.SyncConfigMapWithLabels(
+			ctx,
+			c.configMapGetter,
+			recorder,
+			c.targetNamespace,
+			cm.Name,
+			c.targetNamespace,
+			nameFor(cm.Name, revision),
+			ownerRefs,
+			map[string]string{"operator.openshift.io/controller-instance-name": c.controllerInstanceName},
```

Comment:
nice.  I think this may be unique enough.  We'll need a thing that labels existing revisions to handle an upgrade case too.